### PR TITLE
Estonian translations

### PR DESCRIPTION
**Pull request by hankur**
I didn't translate longer texts (yet), but the interface should be fully translated to Estonian.

### CODE REVIEW

File: app/src/main/res/values-et/strings.xml [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+    <string name="send_address_invalid">See pole sobiv aadress</string>
+    <string name="send_address_not_openalias">OpenAlias aadress pole saadaval</string>
+    <string name="send_address_openalias">OpenAlias secure &#x2714;</string>
+    <string name="send_address_resolve_openalias">Lahendan OpenAlias&#8230;</string>
+    <string name="send_address_no_dnssec">OpenAlias ilma DNSSEC\'ita - aadress võib olla võltsitud</string>
+    <string name="send_title">Saada</string>
+    <string name="send_available">Kontojääk: %1$s XMR</string>
+    <string name="send_address_title">Aadress</string>
+    <string name="send_amount_title">Kogus</string>
+    <string name="send_settings_title">Seaded</string>
```
Comment (m2049r) [inline app/src/main/res/values-et/strings.xml:243]: `send_settings_title` has been removed in master

**Comment by m2049r** [comment]
@erciccione do we want to merge this or wait for completion?

**Comment by erciccione** [comment]
All strings are translated, but i don't know if he plan to translate also the 'help' and 'about' sections. Are you going to make a release soon?

If it's fine for you to have only the strings translated, it can goes for me. I don't have any estonian speakers in the Localization Workgroup. If you merge this, I can open a post on reddit and a task on taiga asking for an estonian translator. Hopefully we can get one soon, but if you are about to release, it will probably stay like this until next one

**Comment by hankur** [comment]
@erciccione I don't really know when I'll get around to translate the other messages, so if you could find other Estonian speakers, it'd be great. But when I have time, I'll translate some more.

**Comment by m2049r** [comment]
@erciccione new release is coming soon. will merge this into it.

@hankur thanks for this :)

**Comment by m2049r** [comment]
@hankur could you please please translate the 4 new strings from #458 ? they are at the bottom of the strings.xml file + the "Street Mode" section in help.xml (only heading is fine, so that it's consistent.

**Pull request merged**